Jenkinsfile

```
@@ -387,7 +166,8 @@ pipeline {
                 stage('Python Bandit check') {
                     when {
                       beforeAgent true
-                      expression { ! skipStage() }
+                      expression { ! skipStage(stage: 'python-bandit',
+		                               def_val: 'false') }
```

If you had to push again for another reason:
```suggestion
		                                 def_val: 'false') }
```

---

Don't know how I missed that.  This was previously approved and it fixes the lack of pylint so I'd prefer to get this landed ASAP.

---

> Don't know how I missed that. This was previously approved and it fixes the lack of pylint so I'd prefer to get this landed ASAP.

More importantly, since you are overriding the `skipStage()` auto-determination (which I am thinking is too rigid and needs some rethought for per-branch overrides), you need to add the `|| quickFunctional()` to your expression since that is also one of the causes to skip this stage.

```suggestion
		                                 def_val: 'false') ||
		                        quickFunctional() }
```

---

I did wonder about that, but I think it's out of scope for this PR.  It looks like it's about a minute for the bandit check, do we want to complicate this when as you say that whole problem needs looking at again anyway?